packages/ag-charts-community/src/chart/navigator/navigator.ts

```diff
@@ -86,13 +86,11 @@ export class Navigator extends BaseModuleInstance implements ModuleInstance {
 
         ctx.scene.root?.appendChild(this.rs);
 
-        const { PauseType } = ctx.interactionManager;
-        const interactionOpts = { bypassPause: [PauseType.ANIMATION] };
         this.destroyFns.push(
-            ctx.interactionManager.addListener('drag-start', (event) => this.onDragStart(event), interactionOpts),
-            ctx.interactionManager.addListener('drag', (event) => this.onDrag(event), interactionOpts),
-            ctx.interactionManager.addListener('hover', (event) => this.onDrag(event), interactionOpts),
-            ctx.interactionManager.addListener('drag-end', () => this.onDragStop(), interactionOpts),
+            ctx.interactionManager.addListener('drag-start', (event) => this.onDragStart(event)),
+            ctx.interactionManager.addListener('drag', (event) => this.onDrag(event)),
+            ctx.interactionManager.addListener('hover', (event) => this.onDrag(event)),
+            ctx.interactionManager.addListener('drag-end', () => this.onDragStop()),
```

Comment:
Can someone clarify what's the intended behaviour of the navigator? Do we want this listener to receive drag events when an animation is in progress?